package.json

```diff
@@ -83,7 +83,7 @@
     "grunt-webpack": "3.1.3",
     "jsdom": "9.12.0",
     "jsdom-global": "2.1.1",
-    "load-grunt-config": "1.0.2",
+    "load-grunt-config": "^3.0.0",
```

Comment:
Why was this package updated?

Reply:
there was a high vulnerability and my hand was itchy to update it 😂 . will revert it for now

Reply:
No worries. No need to revert, just trace it to the places it's used and just make sure no build process is negatively affected. If it's too much to trace and identify then you can revert. But I believe @brandon-binary is working on updating packages.